src/main/resources/templates/mdBenefitsFlow/county.html

```diff
@@ -20,7 +20,7 @@
                                         th:replace="~{fragments/inputs/selectOptionPlaceholder :: selectOptionPlaceholder(optionText=#{county.select})}"/>
                                 <th:block th:each="countyEnum : ${T(org.mdbenefits.app.data.enums.Counties).values()}">
                                     <th:block
-                                            th:replace="~{fragments/inputs/selectOption :: selectOption(value=${countyEnum.name()}, optionText=${countyEnum.getDisplayName()})}"/>
+                                            th:replace="~{fragments/inputs/selectOption :: selectOption(value=${countyEnum.getDisplayName()}, optionText=${countyEnum.getDisplayName()})}"/>
```

Comment:
@vraj - not sure how you're getting the enum when it uses the display name as value, so I wonder if the db record you shared is outdated.